ibis/backends/duckdb/__init__.py

```diff
@@ -437,9 +437,29 @@ def read_in_memory(
         ir.Table
             The just-registered table
         """
+
+        def _clean_up_string_columns(
```

Comment:
Can you add a comment somewhere about what this is working around, and noting when it can be removed?

Reply:
Yup

Reply:
Added a docstring.